websites/M/MangaHere/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "MangaHere",
+    "author": {
+      "name": "Harold",
+      "id": "141206144264699904"
+    },
+    "category": "anime",
+    "logo": "https://i.imgur.com/w7zc3W4.png",
```

Comment:
Not really necessary, but the quality of that logo is just...

Reply:
Ugh, I wanted to recreate it, but it's too much work, they just don't have HD logo..

Reply:
Just resize their logo to 512x512 ty.

Reply:
Yea try getting a better logo